apps/condo/domains/acquiring/schema/AcquiringIntegration.js

```diff
@@ -79,6 +79,10 @@ const AcquiringIntegration = new GQLListSchema('AcquiringIntegration', {
             ...POSITIVE_MONEY_AMOUNT_FIELD,
             schemaDoc: 'The minimum payment amount that can be accepted',
         },
+        maximumPaymentAmount: {
+            ...POSITIVE_MONEY_AMOUNT_FIELD,
```

Comment:
You need to allow null value
Case: We set maximum payment amount, then we want to remove maximum payment amount
You cannot do it through admin ui